nengo/dists.py

```diff
@@ -221,18 +221,32 @@ class UniformHypersphere(Distribution):
     surface : bool, optional (Default: False)
         Whether sample points should be distributed uniformly
         over the surface of the hyperphere (True),
-        or within the hypersphere (False).
+        or within the hypersphere (False). Defaults to False.
+    min_magnitude : Number, optional (Default: 0)
+        Lower bound on the returned vector magnitudes (i.e.
+        magnitude >= min_magnitude). Must be in the range [0, 1]; defaults
+        to 0. Values outside this range will be clipped to this range. Ignored
```

Comment:
I think this description is not entirely correct. If values outside of the range get clipped, I understand that values get sampled in the range [0, 1] and then everything below `min_magnitude` will set to `min_magnitude` (which will cluster vectors at `min_magnitude`). But as I understand the code values are sampled from `[min_magnitude, 1]` and thus are without clustering at `min_magnitude`.